docs/deployment/config_descriptions.md

```diff
@@ -90,36 +90,112 @@ For each service, a TaskCountLowAlarm alarm is added to check the RUNNING Task C
-  if both CPU and memory are under, the EC2 cluster will be scaled in
-* `ecs.autoscaling.scaleOutThresholdPercent`: the reservation percentage where,
-  if both CPU and memory are under, the EC2 cluster will be scaled out
-* `ecs.autoscaling.scalingAdjustmentPercent`: the percentage to increase or
-  decrease the cluster size by when a scaling action is taken. For more
-  information see the
+* `ecs.clusterAutoscaling.scaleInThresholdPercent`: the reservation percentage
+  where, if both CPU and memory are under, the EC2 cluster will be scaled in
+* `ecs.clusterAutoscaling.scaleInAdjustmentPercent`: the percentage to increase
+  or decrease the number of EC2 instances in the cluster when the "scale in"
+  alarm is triggered. Since this is a "scale in" setting, it should typically be
+  a negative value. For more information see the
+  [PercentChangeInCapacity documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-scaling-simple-step.html#as-scaling-adjustment),
+  specifically the section on `PercentChangeInCapacity`.
+* `ecs.clusterAutoscaling.scaleOutThresholdPercent`: the reservation percentage
+  where, if both CPU and memory are under, the EC2 cluster will be scaled out
+* `ecs.clusterAutoscaling.scaleOutAdjustmentPercent`: the percentage to increase
+  or decrease the number of EC2 instances in the cluster when the "scale out"
+  alarm is triggered. Since this is a "scale out" setting, it should typically
+  be a positive value. For more information see the
   [PercentChangeInCapacity documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-scaling-simple-step.html#as-scaling-adjustment),
   specifically the section on `PercentChangeInCapacity`.
 
 ```yaml
 # Defaults
 ecs:
-  autoscaling:
+  clusterAutoscaling:
     scaleInThresholdPercent: 25
+    scaleInAdjustmentPercent: -5
     scaleOutThresholdPercent: 75
-    scalingAdjustmentPercent: 10
+    scaleOutAdjustmentPercent: 10
 ```
 
 The default behavior is that, if more than 75% of your cluster's CPU or memory
 has been reserved, the size of the cluster will be increased by 10%. (There is a
 minimum change of 1 instance.) If _both_ CPU and memory reservation for the
-cluster are under 25%, then the cluster size will be reduced by 10%.
+cluster are under 25%, then the cluster size will be reduced by 5%.
+
+### Service AutoScaling
+
+Cumulus supports automatically scaling the number of tasks configured for an ECS
+service. The scaling of tasks is based on the `ActivityScheduleTime` metric,
+which measures how long (in milliseconds) an activity waited before being picked
+up for processing. If the average activity is waiting more than the configured
+`scaleOutActivityScheduleTime` time, then additional tasks will be added to the
+service. If the average activity is waiting less than the configured
+`scaleInActivityScheduleTime` time, then tasks will be removed from the service.
+Ideally, the average wait time for tasks should settle somewhere between those
```

Comment:
In the pursuit of being _too_ clear, I would recommend:
```
Ideally, the average wait time for tasks should settle somewhere between `scaleOutActivityScheduleTime` and `scaleInActivityScheduleTime`.
```

Reply:
Fixed in 8b60ce787